nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -4,10 +4,12 @@
   pkgs,
   ...
 }:
-
-with lib;
-
 let
+  inherit (lib)
```

Comment:
I did use inherit ... because if i change mkIf and mkMerge with lib. prefix a lot of lines are touched by the nixfmt tool ...